src/OV/OpenVarioMenu.cpp

```diff
@@ -227,16 +934,16 @@ SystemMenuWidget::Prepare([[maybe_unused]] ContainerWindow &parent,
                      "Update System", argv);
   });
 
-  AddButton("Update Maps", [](){
-    static constexpr const char *argv[] = {
-      "/usr/bin/update-maps.sh", nullptr
-    };
+   AddButton("Update Maps", [](){
+     static constexpr const char *argv[] = {
+       "/usr/bin/update-maps.sh", nullptr
+     };
 
-    RunProcessDialog(UIGlobals::GetMainWindow(),
-                     UIGlobals::GetDialogLook(),
-                     "Update Maps", argv);
+     RunProcessDialog(UIGlobals::GetMainWindow(),
+                      UIGlobals::GetDialogLook(),
+                      "Update Maps", argv);
   });
-
+   
```

Comment:
Lots and lots of whitespace changes

Reply:
Unnecessary spaces as in line 946 have been eliminated

Reply:
The other lines as well. You can use https://pre-commit.com/ as a githook. Here is an example file: https://github.com/XCSoar/mapgen/blob/master/.pre-commit-config.yaml
It will check your commits and eliminate white space etc. before committing.